commands/describe.go

```diff
@@ -109,9 +109,16 @@ func runDescribe(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getFunctionURLs(gateway string, functionName string) (string, string) {
+func getFunctionURLs(gateway string, functionName string, functionNamespace string) (string, string) {
 	gateway = strings.TrimRight(gateway, "/")
-	return gateway + "/function/" + functionName, gateway + "/async-function/" + functionName
+	url := gateway + "/function/" + functionName + "." + functionNamespace
```

Comment:
What happens here when the namespace is `""`?

Reply:
The URLs end up having a dot at the end if namespace has nil value.